common/client-libs/validator-client/src/nyxd/error.rs

```diff
@@ -22,7 +22,7 @@ use std::{io, time::Duration};
 #[derive(Debug, Error)]
 pub enum NyxdError {
     #[error("No contract address is available to perform the call")]
-    NoContractAddressAvailable,
+    NoContractAddressAvailable(String),
```

Comment:
If you're storing contract address in the error enum, you might as well return it in the error message